src/extension.ts

```diff
@@ -342,13 +346,6 @@ export async function activate(context: vscode.ExtensionContext) {
   });
   context.subscriptions.push(sdkDeleteWatchDisposable);
 
-  vscode.window.onDidCloseTerminal((terminal: vscode.Terminal) => {
-    terminal.dispose();
-    setTimeout(() => {
-      monitorTerminal = undefined;
-    }, 200);
-  });
-
```

Comment:
Any reason for this code removal?

Reply:
Well I just noticed this code and saw it is redundant. Calling terminal.dispose on OnDidCloseTerminal seems like is not necessary and also monitorTerminal is set as undefined for any terminal being close, which is incorrect.